src/app/components/ImageInput/ImageInput.module.css

```diff
@@ -1,13 +1,8 @@
 body {
   background: var(--color-secondary);
-  color: var(--color-primary);
 }
 
-.container {
-  display: grid;
-}
-
-.container img {
+.image {
   width: 10rem;
   height: auto;
```

Comment:
Is this line really needed? I thought height would automatically adjust, if width is set.